config/locales/en.yml

```diff
@@ -333,6 +333,7 @@ en:
         meta_description: Meta Description
         front_page: Front Page
         active: Active
+        public: Public Category Content
```

Comment:
i18n should be added to all language files as comments to mark untranslated keys